cmd/expiration-mailer/main.go

```diff
@@ -240,6 +240,7 @@ func (m *mailer) processCerts(allCerts []core.Certificate) {
 
 		err = m.sendNags(*reg.Contact, parsedCerts)
 		if err != nil {
+			m.stats.errorCount.With(prometheus.Labels{"type": "sendNags"}).Inc()
```

Comment:
The other `errorCount` increment labels use casing like [`GetRegistration`](https://github.com/lucasallan/boulder/blob/a7a2eaf035554ac10895575d6ab0a177c9a8a2f6/cmd/expiration-mailer/main.go#L202), or [`UpdateCertificateStatus`](https://github.com/lucasallan/boulder/blob/a7a2eaf035554ac10895575d6ab0a177c9a8a2f6/cmd/expiration-mailer/main.go#L224). Could you update this instance to match? E.g. `"SendNags"`?

Reply:
Done.